src/MDMTransition.h

```diff
@@ -37,6 +37,17 @@ NS_SWIFT_NAME(Transition)
 @end
 
 /**
+ Transition with custom duration overrides the default transition duration
```

Comment:
"A transition with custom duration is able to override the default transition duration."